### PR TITLE
Recipe for pairing

### PR DESCRIPTION
**Pull request by haplesshero13**
New post on pairing :)

### CODE REVIEW

**Comment by osis** [comment]
@haplesshero13 This looks great. Out of curiosity, where the the comments come from?

**Comment by haplesshero13** [comment]
They're all fake comments. We made the whole thing up :)

File: content/post/recipe-for-successful-pair-programming.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+figcaption {
+  font-style: italic;
+}
+
+</style>
+
+{{< responsive-figure src="/images/recipe-for-successful-pair-programming/ayen-ybai-pairing.jpeg" class="center small" caption="The authors pairing on this blog post" >}}
+
+Pairing isn't just putting two programmers in front of a computer. This recipe outlines some basic ingredients and the secret sauce for a successful, productive pair programming session.
```
Comment (osis) [inline content/post/recipe-for-successful-pair-programming.md]: My two cents, I think this flows a little bit better if the this statement is above the picture. Looks great though!

Comment (haplesshero13) [inline content/post/recipe-for-successful-pair-programming.md]: I buy that. I'll try it the other way.

**Comment by osis** [comment]
Awesome! Let's do this.

**Comment by moonmaster9000** [comment]
Love it!

**Pull request merged**